vmm/src/device_manager.rs

```diff
@@ -3017,7 +3030,10 @@ impl DeviceManager {
         self.cmdline_additions.as_slice()
     }
 
-    pub fn update_memory(&self, _new_region: &Arc<GuestRegionMmap>) -> DeviceManagerResult<()> {
+    pub fn update_memory_new_region(
```

Comment:
I think naming this function `update_virtio_devices_memory` and have the acpi_resize() all that and then explicitly call the VFIO version is better than two similar entry points into the DeviceManager for two different code paths,

Reply:
Sounds good!